### PR TITLE
Iain/nouns coasters

### PR DESCRIPTION
**Pull request by iainnash**

### CODE REVIEW

File: src/nouns-coasters/utils/InflateLib.sol [hunk 1]
```diff
@@ -0,0 +1,799 @@
...
+pragma solidity >=0.8.0 <0.9.0;
+
+//
+// inflate content script:
+// var pako = require('pako')
+// var deflate = (str) => [str.length,Buffer.from(pako.deflateRaw(Buffer.from(str, 'utf-8'), {level: 9})).toString('hex')]
+//
+
+/// @notice Based on https://github.com/madler/zlib/blob/master/contrib/puff
+library InflateLib {
```
Comment (solimander) [inline src/nouns-coasters/utils/InflateLib.sol:11]: Not sure of the read gas limits that NFT marketplaces impose these days, but [this inflate lib](https://github.com/nounsDAO/nouns-monorepo/blob/master/packages/nouns-contracts/contracts/libs/Inflate.sol) will reduce `tokenURI` gas by roughly 50%. Nouns has used this implementation in prod for roughly a year.

**Comment by iainnash** [comment]
I’ll switch it out!

File: src/distributed-graphics-editions/DistributedGraphicsEdition.sol [hunk 2]
```diff
@@ -1,16 +1,16 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.13;
 
-/**
```
Comment (tbtstl) [inline src/distributed-graphics-editions/DistributedGraphicsEdition.sol:4]: the tiniest of nits

Comment (iainnash) [reply src/distributed-graphics-editions/DistributedGraphicsEdition.sol:4]: i can explain myself: this fixes foundry/forge autodoc feature which was useful. removing this makes the comment not get parsed by docs tools.

**Pull request merged**